### ISSUE TITLE
x1.0 pretrained model

### ISSUE THREAD

**Issue by ArsenLuca** [Randl/ShuffleNetV2-pytorch#2]
Would you please share the shufflenet v2 x1.0 pretrained model? @Randl
